Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'jquery-turbolinks'
 gem 'newrelic_rpm'
 gem 'omniauth-myusa', git: 'https://github.com/18F/omniauth-myusa.git'
 gem 'paperclip'
+gem 'paper_trail', '~> 4.0.0.rc'
```

Comment:
@afeld asked me before if the specific version number was needed, and if so, to add a comment explaining. Now, you too have this request.